scripts/generateToc.js

```diff
@@ -0,0 +1,116 @@
+const path = require('path')
+const fs = require('fs')
+
+const MAIN_HEADER = 'Edge Development Conventions'
+const TOC_FILE_NAME = 'README.md'
+
+const excludedArray = fs
+  .readFileSync(path.join(process.cwd(), '.gitignore'))
+  .toString()
+  .split('\n')
+  .filter(str => str && !str.includes('#'))
+  .map(str => str.replace('/', ''))
+
+const createFolderTree = (
+  startPath = '.',
+  include = '.md',
+  excludes = excludedArray
+) => {
+  const result = { files: [], folders: {} }
+  if (!fs.existsSync(startPath)) return result
+
+  const filePaths = fs.readdirSync(startPath)
+
+  for (const fileName of filePaths) {
+    const fullFilePath = path.join(startPath, fileName)
+
+    if (excludedArray.includes(fullFilePath)) continue
+
+    if (fs.lstatSync(fullFilePath).isDirectory()) {
+      const child = createFolderTree(fullFilePath, include, excludes)
+      if (child.files.length > 0 || Object.keys(child.folders).length > 0) {
+        result.folders[fileName] = child
+      }
+    }
+
+    if (fullFilePath.indexOf(include) >= 0) {
+      const dirname = startPath.substring(startPath.lastIndexOf('/') + 1)
+      if (fileName.split('.')[0] !== dirname) result.files.push(fileName)
+    }
+  }
+
+  return result
+}
+
+const createContentMap = (contentTree, startPath = '.') => {
+  const { files = [], folders } = contentTree
+  const content = []
+  const result = { [startPath]: content }
+
+  for (const file of files) {
+    if (file === TOC_FILE_NAME) continue
+
+    const markdownFile = fs.readFileSync(path.join(startPath, `${file}`))
+    const title = markdownFile.toString().split('\n')[0].split(' &nbsp; ')[1]
```

Comment:
Why are you taking the first line of the file and the second part of the line after " &nbsp; "?